web/templates/settings/profile_settings.hbs

```diff
@@ -6,7 +6,7 @@
                 <div class="full-name-change-container">
                     <div class="input-group inline-block grid user-name-parent">
                         <div class="user-name-section inline-block">
-                            <label for="full_name" class="settings-field-label inline-block">{{t "Name" }}</label>
+                            <label {{#if user_can_change_name}}for="full_name"{{/if}} class="settings-field-label inline-block">{{t "Name" }}</label>
```

Comment:
Why is this `if` block needed? We do not do this for other disabled fields like when non-admin user opens organization settings.

Reply:
Yeah that is unnecessary. I originally wrote that because I noticed that the input could be disabled and thought that by removing the `for` attribute, the cursor for the label would remain as the default cursor. Which wasn't the case.